frontend/src/app/spot/components/text-field/stories/TextField.stories.mdx

```diff
@@ -104,7 +104,7 @@ The **Default** state indicates that a text field is ready to accept text input.
 
 The **Focus** state has a coloured outline, the same as any focused element in OpenProject.
 
-The **Hover** element has a darker border.
+The **Hover** element has a darker border (Basic/Gray 1: #333333).
```

Comment:
I'd refrain from using hex codes here, it just means you'll have to update more docs if you ever update the value, and for daily usage both in figma and in code it's irrelevant anyway

Reply:
Removed hex.